### PR TITLE
Cleaned up a bit of the wording, numbering, spacing.

### PR DESCRIPTION
**Pull request by kpacquer**

### CODE REVIEW

File: en-US/Docs/Store.md [hunk 1]
```diff
@@ -1,55 +1,66 @@
...
 lang: en-US
 ---
 
 # Installing and Servicing apps on Windows 10 IoT Core
 
-Microsoft makes it easy for OEMs to install and service UWP apps on Windows 10 IoT Core through the Universal Store. All store signed apps on Windows 10 devices are capable of receiving updates directly from the store. 
+Developers can install and service Universal Windows Platform (UWP) apps on Windows 10 IoT Core through the Windows Store. All Store-signed apps on Windows 10 devices are capable of receiving updates directly from the Windows Store. 
 
-{% include note.html text="Preinstall and servicing of store signed apps is only available to Windows 10 IoT Core Pro customers. If you are an OEM wishing to preinstall apps on your IoT devices please visit our [Commercialization](https://www.windowsforiotdevices.com) page to get started. If you are developer wishing to preinstall apps for IoT you must seek sponsorship from an OEM enrolled in the program." %}
+To submit apps and service them through the Windows Store, you'll need to join the Windows Store OEM Preinstall Program. To get enrolled, contact your Microsoft Channel Partner or [partnerops@microsoft.com](mailto:partnerops@microsoft.com)
```
Comment (trmck) [inline en-US/Docs/Store.md:14]: @kpacquer This section here needs to convey a message similar to below. 

"Preinstall and servicing of store signed apps is only available to Windows 10 IoT Core Pro customers. If you are an OEM wishing to preinstall apps on your IoT devices please visit our [Commercialization](https://www.windowsforiotdevices.com) page to get started. If you are developer wishing to preinstall apps for IoT you must seek sponsorship from an OEM enrolled in the program."

**Comment by ivorb** [comment]
please sync with the remote so we can auto merge this.  thanks!

**Comment by ivorb** [comment]
@kpacquer ping

**Review by kpacquer — COMMENTED** [review]
Reviewed

**Pull request closed without merging**